src/components/datepicker/calendar.js

```diff
@@ -58,6 +58,7 @@
       scope: {
         minDate: '=mdMinDate',
         maxDate: '=mdMaxDate',
+        isDateEnabled: '=mdDateEnabled',
```

Comment:
@ThomasBurleson what do you think of calling this `md-date-filter` ?

(I won't go as far as suggesting `md-date-predicate`...)